PubnativeLite/PubnativeLite/Consent/HyBidUserDataManager.m

```diff
@@ -70,7 +70,7 @@ + (instancetype)sharedInstance {
     return sharedInstance;
 }
 
-- (void)createUserDataManagerWithAppToken:(NSString *)appToken completion:(UserDataManagerCompletionBlock)completion {
```

Comment:
Why do we remove this param? Was it not used?

Reply:
We are using the appToken that is saved to the HyBid sharedInstance